content/Software and Downloads/Installation/Add-a-library-to-Arduino-IDE.md

```diff
@@ -0,0 +1,61 @@
+---
+title: "Add a library to Arduino IDE"
+---
+
+Libraries are a collection of code that makes it easy to connect sensors, RTCs, Wi-Fi modules, RGB matrices and other components to your board.
+
+---
+
+## Add a library in Library manager
+
+1. In the menu bar, go to `Tools > Manage Libraries...` (In IDE 2, you can also click on the Library Manager icon on the left).
+
+2. Search for the library you want to install.
+
+3. Find the library in the search results.
+
+4. Click on the INSTALL button.
+
+   ![IDE 1 Library Manager with search box and Install button highlighted](img/LibraryManager_InstallingLibrary_IDE1.png)
+
+   ![IDE 2 Library Manager Icon, search box, and Install button highlighted](img/LibraryManager_InstallingLibrary_IDE2.png)
+
+5. Wait for the installation to complete.
+
+6. After the installation is complete, the library will be labelled as INSTALLED, and it is ready to be [included in a sketch](https://www.arduino.cc/reference/en/language/structure/further-syntax/include/).
+
+---
+
+## Importing a .zip Library
+
+1. In the menu bar, go to `Sketch > Include Library > Add .ZIP Library...`
+
+2. You will be prompted to select the library you want to add. Navigate to the .zip file's location and open it.
+
+3. The library will automatically be installed and ready to be [included in a sketch](https://www.arduino.cc/reference/en/language/structure/further-syntax/include/).
+
+## Manual Installation
+
+---
+
+1. Go to the directory where you have downloaded the .zip library file.
+
+2. Extract the .zip file.
+
+3. Select the main folder (it should have the library's name) and move it to the "libraries" folder [inside your sketchbook](https://support.arduino.cc/hc/en-us/articles/4412950938514-Open-the-Sketchbook).
+
+4. The library will automatically be installed and ready to be [included in a sketch](https://www.arduino.cc/reference/en/language/structure/further-syntax/include/).
```

Comment:
With this installation method, the library won't be available until the IDE is restarted. I would add restarting Arduino IDE as a step:

```suggestion
4. Restart Arduino IDE.
```